integrations/airflow/documentation.yaml

```diff
@@ -0,0 +1,69 @@
+exporter_type: included
+app_name_short: Airflow
+app_name: Apache {{app_name_short}}
+app_site_name: Airflow
+app_site_url: https://airflow.apache.org/
+exporter_name: the Airflow exporter
+exporter_pkg_name: airflow
+exporter_repo_url: https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/metrics.html
+additional_install_info: |
+  To determine if {{exporter_name}} is exposing metrics, access the metrics endpoint
+  by using port forwarding and curl as follows:
+
+  In one terminal, configure port forwarding by using `kubectl -n <namespace_name> port-forward <airflow_pod_name> 9102`.
+  The port forwarding command allows access to port 9102 on your machine.
+
+  In a second terminal, make a `GET` request to the metrics endpoint by using `curl localhost:9102/metrics`.
```

Comment:
@yqlu: yeah, you are probably right about it being "prereq" info, not "install" info! Good catch. (Sorry, Joe!)

No objection to your "consistent-with-Istio" wording suggestion, either.

Reply:
I opted for the deployment name, instead of picking a specific pod:
```
  <pre class="devsite-click-to-copy">
  kubectl -n {{namespace_name}} port-forward deploy/airflow-statsd 9102
  </pre>
```